### PR TITLE
Add mocked Recommended Categories cluster to stream API

### LINKED ISSUE #67: [Backend] Add mocked Recommended Categories cluster to Stream API

### PR DESCRIPTION
**Pull request by essencec**

### CODE REVIEW

File: src/main/java/com/google/sps/models/RecommendedCategories.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        container.add(mockContainer1);
+        container.add(mockContainer2);
+        container.add(mockContainer3);
+        container.add(mockContainer4);
+        container.add(mockContainer5);
+        container.add(mockContainer6);
+        container.add(mockContainer7);
+        container.add(mockContainer8);
+        
+
```
Comment (ajwang0) [inline src/main/java/com/google/sps/models/RecommendedCategories.java]: Remove extra new lines

File: src/main/java/com/google/sps/models/RecommendedCategories.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    public String getType() {
+        return "RecommendedCategories";
+    }
+
+    public String getID() {
+        return "recommendedCategoriesID";
+    }
+
+    public void makeCluster() {
+        Container mockContainer1 = new Container("mockContainer1", "Health & fitness", "mockIcon1.png");
```
Comment (ajwang0) [inline src/main/java/com/google/sps/models/RecommendedCategories.java]: I think we should rename this to something more specific. Container is a bit too generic. How about RecommendedCategoryLink or RecommendedCategoryCard

**Review by ajwang0 — DISMISSED** [review]
Don't forget to add it to https://github.com/googleinterns/play-web-step-2020/blob/master/src/main/java/com/google/sps/models/Stream.java

**Comment by essencec** [comment]
> Don't forget to add it to https://github.com/googleinterns/play-web-step-2020/blob/master/src/main/java/com/google/sps/models/Stream.java

I believe it's already there

**Review by tonymai — CHANGES_REQUESTED** [review]
Where are the tests?

**Review by tonymai — APPROVED** [review]
It seems like we haven't written tests for models. I think this is fine right now, in the interest of time and given that there isn't much logic in the models.

**Pull request merged**